pkg/genericcontrolplane/server.go

```diff
@@ -79,9 +92,9 @@ func Run(completeOptions completedServerRunOptions, stopCh <-chan struct{}) erro
 	apiExtensionsConfig, err := CreateAPIExtensionsConfig(
 		*config.GenericConfig,
 		config.ExtraConfig.VersionedInformers,
-		pluginInitializer,
-		completeOptions.ServerRunOptions,
-		serviceResolver,
+		nil,
```

Comment:
```suggestion
		nil, // pluginInitializer
```

Reply:
<img width="161" alt="Bildschirmfoto 2022-02-08 um 23 37 52" src="https://user-images.githubusercontent.com/730123/153087448-3f66c3d9-68fb-420b-91c7-136d7f6b05eb.png">

Goland is clever.

Fixed.